modules/nw-egress-service-ovn.adoc

```diff
@@ -0,0 +1,114 @@
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* You configured MetalLB `BGPPeer` resources.
+// Can we make this part of the procedure?
+
+.Procedure
+
+. Create an `IPAddressPool` custom resource (CR) with the desired IP for the service. 
+
+.. Create a file, such as `ip-addr-pool.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: example-pool
+  namespace: metallb-system
+spec:
+  addresses:
+  - 172.19.0.100/32
+  autoAssign: false <1>
+----
+<1> Set `autoAssign` to `false` to ensure MetaLB reserves IP addresses in the IP address pool exclusively for the `LoadBalancer` service.
+
+.. Apply the configuration for the IP address pool by running the follow command:
++
+[source,terminal]
+----
+$ oc apply -f ip-addr-pool.yaml
+----
+
+. Create `Service` and `EgressService` CRs. 
+
+.. Create a file, such as `service-egress-service.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: example-service
+  namespace: example-namespace
+  annotations:
+    metallb.universe.tf/address-pool: example-pool <1>
+spec:
+  selector:
+    app: example
+  ports:
+    - name: http
+      protocol: TCP
+      port: 8080
+      targetPort: 8080
+  type: LoadBalancer
+---
+apiVersion: k8s.ovn.org/v1
+kind: EgressService
+metadata:
+  name: example-service
+  namespace: example-namespace
+spec:
+  sourceIPBy: "LoadBalancerIP" <2>
+  nodeSelector: <3>
+    matchLabels:
+      node-role.kubernetes.io/worker: "" 
+----
+<1> The `LoadBalancer` service uses the IP address assigned by MetalLB from the `example-pool` IP address pool. 
+<2> Select `LoadBalancerIP` to assign the ingress IP address of the `LoadBalancer` service as the source IP address of egress traffic.
+<3> Select a node to handle the `LoadBalancer` service's traffic. In this example, the following label will be applied to the node to identify it as the node that handles service traffic: `egress-service.k8s.ovn.org/example-namespace-example-service: ""`.
+
+.. Apply the configuration for the service and egress service by running the follow command:
++
+[source,terminal]
+----
+$ oc apply -f service-egress-service.yaml
+----
+
+. Create a `BGPAdvertisement` CR to advertise the service.
+
+.. Create a file, such as `service-bgp-advertisement.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: BGPAdvertisement
+metadata:
+  name: example-bgp-adv
+  namespace: metallb-system
+spec:
+  ipAddressPools:
+  - example-pool
+  nodeSelector:
+  - matchLabels:
+      egress-service.k8s.ovn.org/example-namespace-example-service: "" <1>
+----
+<1> To ensure ingress replies to egress traffic can return with a known source IP address, select the node that handles the `LoadBalancer` service.
```

Comment:
`with a known source IP address` is irrelevant, `To ensure ingress replies to egress traffic from the service's pods are handled properly` instead should be enough imo.
the other part should be emphasized - only the node that handles the service should advertise it

Reply:
I changed "handled properly" to "routed correctly"